controller/deis/settings.py

```diff
@@ -290,6 +290,9 @@
 # names which apps cannot reserve for routing
 DEIS_RESERVED_NAMES = ['deis']
 
+# the default expected status code for healthchecks
+DEIS_HEALTHCHECK_STATUS_CODE = '200'
```

Comment:
Should we also put this in the confd template for the settings so it can be overridden by setting an etcd key (like `/deis/controller/healthcheckDefaultStatusCode`)?